crates/aptos-crypto/src/arkworks/shamir.rs

```diff
@@ -161,11 +185,25 @@ impl<F: PrimeField> ThresholdConfig<F> {
     /// 1. A random polynomial of degree `t-1` is generated with `val_to_share`
     ///    as the constant term.
     /// 2. The polynomial is evaluated over the `domain` using FFT to produce `y` values.
-    /// 3. Each share is represented as a `(x, y)` pair (`ShamirShare<F>`).
+    /// 3. Each share is represented as a `(x, y)` pair (`ShamirShare<F>`).  TODO: not used atm
     pub fn share<R: Rng + RngCore>(&self, val_to_share: F, rng: &mut R) -> Vec<ShamirShare<F>> {
         let mut coeffs = vec![val_to_share]; // constant term of polynomial
         coeffs.extend((0..(self.t - 1)).map(|_| F::rand(rng)));
-        let y_pts = self.domain.fft(&coeffs);
+
+        self.share_with_coeffs(&coeffs)
```

Comment:
Don't understand why all this indirection was needed: `share_with_coeffs` --> `share_with_coeffs_only_evals`

Reply:
Let's not guess what Rex needs for now. Let's write the minimal PR we need.